src/CMakeLists.txt

```diff
@@ -1,22 +1,34 @@
 # Filling templates with shaders code.
 file(READ simple.vert SIMPLE_VERT)
 file(READ simple.frag SIMPLE_FRAG)
-configure_file(shaders.cpp.in shaders.cpp)
-configure_file(shaders.hpp ${CMAKE_CURRENT_BINARY_DIR})
+configure_file(shaders.cpp.in shaders.cpp @ONLY)
 
 add_executable(
     hello_gl
     shaders.hpp
-    ${CMAKE_CURRENT_BINARY_DIR}/shaders.cpp
     main.cpp
-    )
+    ${CMAKE_CURRENT_BINARY_DIR}/shaders.cpp
+  )
 
-target_include_directories(hello_gl PRIVATE "${GLAD_DIR}/include")
+target_compile_options(
+    hello_gl
+    PRIVATE
+        -Wall
+        -Wextra
+  )
+
+target_include_directories(
+    hello_gl
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}
+  )
 
 target_link_libraries(
     hello_gl
     PRIVATE
-        glad
-        ${OPENGL_LIBRARY}
```

Comment:
For every third party library (and CMake finder module) use an imported library if provided. If it doesn't, make an imported library and use it. **Do not use variables!**